conda_recipes/cinema4d-openjd/recipe/activate.bat

```diff
@@ -0,0 +1,13 @@
+set DEADLINE_CLOUD_PYTHONPATH="%%CONDA_PREFIX%%\Lib\site-packages"
+set DEADLINE_CINEMA4D_EXE="%%CONDA_PREFIX%%\c4d\Commandline.exe"
+
+REM We need to add additional paths for pywin32 dependency.
+set C4DPYTHONPATH311="%%CONDA_PREFIX%%\Lib\site-packages;%%CONDA_PREFIX%%\Lib\site-packages\win32;%%CONDA_PREFIX%%\Lib\site-packages\win32\lib;%%CONDA_PREFIX%%\Lib\site-packages\Pythonwin"
+
+REM You can add your license environment variables for Cinema 4D and Redshift here. 
+REM For licensing Cinema 4D, you can use the g_licenseServerUrl variable. 
+REM https://support.maxon.net/hc/en-us/articles/1500006333301-R21-License-Server-for-R21-and-onward-automatic-user-login
+REM set g_licenseServerUrl=http://[ip address]:[port]
+REM For licensing Redshift, you can use the redshift_LICENSE environment variable. 
+REM https://help.maxon.net/c4d/en-us/#../Subsystems/Default/Content/html/RLM+Floating+Licenses.html#RLMFloatingLicenses-Option1-EnvironmentVariables
+REM set redshift_LICENSE=[port]@[ip-address]
```

Comment:
I don't think this block comment should be here. As a general rule, mixing configuration like license servers and ports with software packages is not a good structure to use. Something like queue environments (https://docs.aws.amazon.com/deadline-cloud/latest/developerguide/configure-jobs.html) is better suited to the task.

Reply:
Makes sense, it was simpler to use the license environment variables in my testing but queue environments definitely looks better suited. I'll update this.